core/deployment/src/main/java/io/quarkus/deployment/builditem/DevServicesSharedNetworkBuildItem.java

```diff
@@ -8,16 +8,16 @@
 
 import io.quarkus.builder.BuildChainBuilder;
 import io.quarkus.builder.BuildStepBuilder;
-import io.quarkus.builder.item.SimpleBuildItem;
+import io.quarkus.builder.item.MultiBuildItem;
 import io.quarkus.deployment.pkg.builditem.ProcessInheritIODisabled;
 
 /**
- * A marker build item that if present during the build, then the containers started by DevServices
+ * A marker build item that, if any instances are provided during the build, the containers started by DevServices
  * will use a shared network.
  * This is mainly useful in integration tests where the application container needs to be able
  * to communicate with the services containers
  */
-public final class DevServicesSharedNetworkBuildItem extends SimpleBuildItem {
+public final class DevServicesSharedNetworkBuildItem extends MultiBuildItem {
```

Comment:
I don't think this is correct, what's the point of producing multiple DevServicesSharedNetworkBuildItem? Isn't just a simple flag?

Reply:
It is correct, since it can be generated by multiple extensions theoretically

Reply:
@gastaldi I have written an extension that launches various containers and needs them all on the shared network. After some discussion it was agreed the existing mechanism with `DevServicesSharedNetworkBuildItem` should be used by my extension. In theory, when I write an integration test for my extension, Quarkus will also produce a `DevServicesSharedNetworkBuildItem` to setup the IT environment.. meaning we'd have two producers.